dev/ci/ci-tlc.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download tlc
+
+( cd "${CI_BUILD_DIR}/tlc" && make "-j${NJOBS}" && make install )
```

Comment:
```suggestion
( cd "${CI_BUILD_DIR}/tlc" && make && make install )
```

Reply:
I'll change. I copied this line from Flocq and Coquelicot. Maybe you want to fix them too.

Reply:
Indeed, I think as of today the `-j` flag is not needed as we pass it to the top-level CI makefile, right @SkySkimmer ?

Reply:
>I'll change. I copied this line from Flocq and Coquelicot. Maybe you want to fix them too.

Those use remake, not make ;)